package.json

```diff
@@ -2,7 +2,7 @@
   "name": "innoq-styleguide",
   "version": "0.7.7",
   "description": "INNOQ's Styleguide and Component Library",
-  "license": "SEE LICENSE IN <LICENSE.md>",
```

Comment:
AFAIK you have to set it to `SEE LICENSE IN <…>` as npm parses that in order to hide the license warning in CLI.

Reply:
[TIL](https://docs.npmjs.com/files/package.json#license) - in that case, please disregard bf4bc10a3c7f686363e52ad66bc62d36fd88b7f0 (I'm AFK so cannot push myself right now)